odoo/addons/base/models/ir_http.py

```diff
@@ -487,3 +490,34 @@ def _response_by_status(self, status, headers, content):
             return werkzeug.utils.redirect(content, code=301)
         elif status != 200:
             return request.not_found()
+
+    @classmethod
+    def _get_session_store(cls):
+        path = odoo.tools.config.session_dir
+        _logger.debug('HTTP sessions stored in: %s', path)
+        return sessions.FilesystemSessionStore(
+            path, session_class=OpenERPSession, renew_missing=True)
+
+    @api.autovacuum
+    def _session_vacuum(self):
+        """
+        Remove all session file unused for some time based
+        on the ir.config_parameter base.unused_session_age_limit
+
+        Or default to one week in seconds.
+        """
+        Param = self.env['ir.config_parameter']
+        session_store = self._get_session_store()
+
+        session_age_limit = Param.sudo().get_param(
+            'base.unused_session_age_limit',
+            60*60*24*7
+        )
+
+        last_week = time.time() - session_age_limit
+
+        session_path = Path(session_store.path)
+
+        for session_file in session_path.glob('*'):
```

Comment:
`iterdir()`? `glob('*')`  just seems like a more complicated version since we're not actually filtering anything out.

Reply:
Yes okay, I first thought about using `werkzeug_*.sess` but I believe the format of session file changed between odoo version. I guess iterdir should be fine.